util/rate_tracker.h

```diff
@@ -0,0 +1,119 @@
+//  Copyright (c) Meta Platforms, Inc. and affiliates.
+//  This source code is licensed under both the GPLv2 (found in the
+//  COPYING file in the root directory) and Apache 2.0 License
+//  (found in the LICENSE.Apache file in the root directory).
+//
+
+#pragma once
+
+#if defined(_WIN32)
+#else
+// Unix/Linux-specific headers
+#include <sys/resource.h>
+#endif
+
+#include <atomic>
+
+#include "rocksdb/options.h"
+#include "rocksdb/rate_limiter.h"
+#include "rocksdb/system_clock.h"
+
+namespace ROCKSDB_NAMESPACE {
+
+// RateTracker is a template class that tracks the rate of change of
+// values over time (result returned unit/s). It records data points with
```

Comment:
// (result returned unit/s) 

This is not specific enough. Maybe "The GetRate() method returns the rate in units per second (unit/s)."?